spec.md

```diff
@@ -1290,9 +1308,9 @@ $ curl 'http://username:password@service-broker-url/v2/service_instances/:instan
 | 410 Gone | MUST be returned if the binding does not exist. The expected response body is `{}`. |
 
 Responses with any other status code will be interpreted as a failure and the
-binding will remain in the marketplace database. Service brokers can include a
-user-facing message in the `description` field; for details see [Service
-Broker Errors](#service-broker-errors).
+binding MUST remain in the marketplace database.
```

Comment:
Is this the first time we've used "marketplace database"? Can we rephrase this to something like:
```
Responses with any other status code will be interpreted as a failure and the Platform MUST continue to remember the Service Binding.
```

Reply:
its not the only time we talk about DBs for the marketplace. Let me see if I can sneak in that change....

Reply:
actually, let me do that in a new PR since its more than just a one word change and might require some thinking by the reviewers.

Reply:
See: https://github.com/openservicebrokerapi/servicebroker/pull/362

Reply:
Thank you! :)